apps/web/src/components/Fields/SmartInput.tsx

```diff
@@ -146,6 +136,18 @@ const SmartInput: React.FC<SmartInputProps> = ({
             : undefined
         }
       />
+
```

Comment:
I think we should leave the error message where it was before unless you have a solid reason for moving it

Reply:
It looked bad!!! and inside the input field